models/spike_recorder.h

```diff
@@ -64,6 +66,8 @@ spike creation rather than that of their arrival.
 The call to ``Connect`` will fail if the connection direction is
 reversed (i.e., connecting *sr* to *neurons*).
 
+.. include:: ../models/recording_device.rst
+
 EndUserDocs */
```

Comment:
Add a See-also section to actually refer to other recorders
```suggestion
.. include:: ../models/recording_device.rst

See also
++++++++

EndUserDocs */
```